code/game/turfs/open/glass.dm

```diff
@@ -27,6 +27,8 @@
 /turf/open/floor/glass/LateInitialize()
 	. = ..()
 	AddElement(/datum/element/turf_z_transparency, TRUE)
+	plane = FLOOR_PLANE
+	layer = TURF_LAYER
```

Comment:
Not changing the plane should be an option attached to the element, rather then being hardcoded like this

Reply:
This has been changed.